Gir.toml

```diff
@@ -50,6 +50,10 @@ status = "generate"
     #manual is_windows_utf8
     ignore = true
     [[object.function]]
+    name = "new"
```

Comment:
Not sure why the other constructors are no nullable. They should probably but are not marked as such in the latest .gir either

Reply:
I will check sources later.